myft/templates/save-for-later.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export default ({ myFtApiWrite, contentId, variant, buttonText }) => {
+	return !myFtApiWrite ? null : (
```

Comment:
Always find negations harder to understand, but that's just me, don't have a string opinion about it